libnest/src/config.rs

```diff
@@ -104,6 +104,23 @@ impl Config {
         &self.download_path
     }
 
+    /// Returns a mutable reference to the path where packages's datas are downloaded.
```

Comment:
See below (about "datas"), there may be other occurrences, but I won't report them.